Serializer/Normalizer/ExceptionHandler.php

```diff
@@ -93,11 +93,11 @@ public function serializeToXml(
     }
 
     /**
-     * @param ExceptionWrapper $exceptionWrapper
+     * @param \Exception $exception
      *
      * @return array
      */
-    protected function convertToArray(ExceptionWrapper $exceptionWrapper)
+    protected function convertToArray(\Exception $exception, Context $context)
     {
         return [
             'code' => $exceptionWrapper->getCode(),
```

Comment:
Yeah you should not look the normalizer inner. 
 I just wanted to have feedbacks on the globality of the proposal so I didn't update everything